### ISSUE TITLE
npm WARN cannot run in wd bcrypt@0.8.5 node-gyp rebuild

### ISSUE THREAD

**Issue by beinginorbit** [iceddev/bcrypt-as-promised#3]
When attempting to install `bcrypt-as-promised` on `nodesource/trusty` Docker container I get the following error:

```
npm WARN cannot run in wd bcrypt@0.8.5 node-gyp rebuild (wd=/usr/src/app/node_modules/bcrypt-as-promised/node_modules/bcrypt)
```

Using bcrypt as normal works fine. I wonder if bcrypt just needs to be upgraded.

**Comment by phated** [comment]
The semver range picks up the latest bcrypt version.  I'm not sure why you are getting a rebuild on install.  Was this a fresh install?  If not, you might need to remove and reinstall from scratch.

**Comment by beinginorbit** [comment]
Yes this is a clean install new build from Docker. Something screwy with npm (0.12.6) maybe. This also happens with `bcrypt-then`.

**Comment by lishoulong** [comment]
I encounter the same problem, "493 error Failed at the bcrypt@0.8.6 install script 'node-gyp rebuild'." I am on the windows 8 platform. And I tried for several times.

**Comment by darkn3rd** [comment]
Trying from Ubuntu 14.04 Trusty Tahr (running using vagrant on OS X) and trying to run this in a container, get it freezes w/o any further information:

```
> bcrypt@0.8.6 install /gobalto/node_modules/bcrypt
> node-gyp rebuild
```

**Comment by TomFreudenberg** [comment]
Maybe you check here https://github.com/npm/npm/issues/8862#issuecomment-136420900, this solved my issue
